contentcuration/contentcuration/static/js/edit_channel/channel_set/views.js

```diff
@@ -0,0 +1,129 @@
+import Vue from 'vue';
+import ChannelSetModalComponent from './views/ChannelSetModal.vue';
+
+var Backbone = require('backbone');
+var BaseViews = require("../views");
+var dialog = require("../utils/dialog");
+var store = require('./vuex/store');
+var vueIntl = require("vue-intl");
+var translations = require("utils/translations");
+var { PageTypes } = require('./constants');
+
+var ChannelSetModal = Vue.extend(ChannelSetModalComponent);
+
+// Flatten translation dictionary
+var unnested_translations = {};
+Object.keys(translations).forEach(function (key) {
+    Object.keys(translations[key]).forEach(function(nestedKey) {
+        unnested_translations[key + "." + nestedKey] = translations[key][nestedKey];
+    });
+});
+
+Vue.use(vueIntl, {"defaultLocale": "en"});
```

Comment:
Might be better to avoid repeating this code all over the code base - could we create a central module that we import Vue from that does this initialization for us?

Reply:
Sure! Where would be the best place to put this? utils/string_helper?